backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -453,6 +453,11 @@ public DeleteRepositoryRequest.Response deleteRepository(
   @Override
   public void deleteRepositories(
       Session session, ExperimentRunDAO experimentRunDAO, List<String> allowedRepositoryIds) {
+    deleteRepositoriesCollection(session, experimentRunDAO, allowedRepositoryIds);
+  }
+
+  public void deleteRepositoriesCollection(
```

Comment:
do we really need both these methods? Can't we just have one of them public?

Reply:
removed one of them